# Упражнение по программированию 15.3  
Это упражнение подобно упражнению 2, но исключения должны быть производными от  
базового класса (потомка `logic_error`), который хранит два значения  
аргументов. Исключения должны содержать метод, который используется для   
обоих исключений. Каждое исключение должно приводить к прекращению цикла  
обработки.  
